src/main.rs

```diff
@@ -25,4 +25,5 @@ fn main() {
     let mut ctx: ClipboardContext = ClipboardProvider::new().unwrap();
 
     ctx.set_contents(output).unwrap(); // Copy the header to clipboard.
+    thread::sleep(time::Duration::from_secs(1)); // This is for Linux users.
```

Comment:
Can be changed to `from_millis(100)` to be faster

Reply:
thanks for the suggestion :)